src/translations/nl_NL.translation

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Translations xmlns="http://soap.sforce.com/2006/04/metadata">
     <customLabels>
-        <label><!-- AutoCreateFieldError --></label>
+        <label>Niet in staat om een geldig Id op te halen voor het automatisch aanmaken van de Relatie. Controleer op geldige velden in de instellingen onder Relatie Instellingen.</label>
         <name>AutoCreateFieldError</name>
     </customLabels>
     <customLabels>
-        <label><!-- Female --></label>
+        <label>Vrouw</label>
         <name>Female</name>
     </customLabels>
     <customLabels>
-        <label><!-- Male --></label>
+        <label>Man</label>
```

Comment:
Not that I speak Dutch, but is this the correct translation for "Male"?

Reply:
It looks like it is the translation for Man but I did cross reference the spreadsheet and that's what is there.